spec.md

```diff
@@ -912,6 +912,21 @@ The following HTTP Headers are defined for this operation:
 | organization_id | string | Deprecated as it was redundant information. Organization for the Service Instance MUST be provided by Platforms in the top-level field `context`. If present, it MUST be the ID of the organization specified for the Service Instance. |
 | space_id | string | Deprecated as it was redundant information. Space for the Service Instance MUST be provided by Platforms in the top-level field `context`. If present, it MUST be the ID of the space specified for the Service Instance. |
 
+Note: The `parameters` specified are expected to be the values specified
+by an end-user. Whether the user chooses to include the complete set of
```

Comment:
>an end-user

Does this imply a person is filling these values out vs an automated system? Does end-user also include a Platform?

Reply:
In CF, configuration parameters are chosen by a person rather than the system.